### ISSUE TITLE
Update code with closed form for number of cells at a given resolution

### ISSUE THREAD

**Issue by ajfriend** [uber/h3#408]
And maybe add something to the comments for that function like:

```
Let h(n) be the number of children n levels below
a single *hexagon*.

Then h(n) = 7^n.

Let p(n) be the number of children n levels below
a single *pentagon*.

Then p(0) = 1, and p(1) = 6, since each pentagon
has 5 hexagonal immediate children and 1 pentagonal
immediate child.

In general, we have the recurrence relation

p(n) = 5*h(n-1) + p(n-1)
     = 5*7^(n-1) + p(n-1).

Working through the recurrence, we get that

p(n) = 1 + 5*\sum_{k=1}^n 7^{k-1}
     = 1 + 5*(7^n - 1)/6,

using the closed form for a geometric series.

Using the closed forms for h(n) and p(n), we can
get a closed form for the total number of cells
at resolution r:

c(r) = 12*p(r) + 110*h(r)
     = 2 + 120*7^r.
```

**Comment by nrabinowitz** [comment]
Trying hard to follow :). Should this part:
```
p(n) = 5*h(n-1) + p(n-1)
     = 5*7^n + p(n-1)
```
read 
```
p(n) = 5*h(n-1) + p(n-1)
     = 5*7^(n - 1) + p(n-1)
```
or am I confused?

**Comment by ajfriend** [comment]
> Trying hard to follow :). Should this part:
> 
> ```
> p(n) = 5*h(n-1) + p(n-1)
>      = 5*7^n + p(n-1)
> ```
> 
> read
> 
> ```
> p(n) = 5*h(n-1) + p(n-1)
>      = 5*7^(n-1) + p(n-1)
> ```
> 
> or am I confused?

Completely correct! That's definitely a typo. I'll update above. Also, this writeup is hiding several whiteboards of work. :)
